packages/frontend/src/components/Explorer/VisualizationCard/VisualizationCard.tsx

```diff
@@ -65,9 +65,11 @@ const VisualizationCard: FC = () => {
         <>
             <Card style={{ padding: 5, overflowY: 'scroll' }} elevation={1}>
                 <VisualizationProvider
-                    chartConfigs={savedChart?.chartConfig}
+                    initialChartConfig={unsavedChartVersion.chartConfig}
```

Comment:
We can now use the unsavedChartVersion since it won't trigger an infinite loop. We need this because we will, in the next PR, load the chart config based on url params.